verticapy/machine_learning/vertica/neighbors.py

```diff
@@ -787,6 +781,8 @@ def _vertica_fit_sql(self) -> Literal["RF_REGRESSOR"]:
     def _vertica_predict_sql(self) -> Literal["PREDICT_RF_REGRESSOR"]:
         return "PREDICT_RF_REGRESSOR"
 
+    # This is an exception. Although KernelDensity is a subclass of Regressor,
+    # but it is UNSUPERVISED.
```

Comment:
@oualib, can you please confirm that this exception is made on purpose?

Reply:
yes